zeebe/exporters/operate-exporter/src/main/java/io/camunda/zeebe/operate/exporter/util/XMLUtil.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.operate.exporter.util;
+
+import io.camunda.operate.entities.ProcessEntity;
+import io.camunda.operate.entities.ProcessFlowNodeEntity;
+import io.camunda.zeebe.model.bpmn.Bpmn;
+import io.camunda.zeebe.model.bpmn.BpmnModelInstance;
+import io.camunda.zeebe.model.bpmn.instance.FlowNode;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.*;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.parsers.SAXParserFactory;
+import org.camunda.bpm.model.xml.ModelException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.xml.sax.Attributes;
+import org.xml.sax.SAXException;
+import org.xml.sax.helpers.DefaultHandler;
+
+public class XMLUtil {
```

Comment:
Do we plan to make this a spring bean later on? This feels like a shared component and passing a single instance to the various handlers rather than having each handler create and manage its own instance would make things easier from a management and testing standpoint